File: cpp codes/code5.cpp
#include <iostream>
using namespace std;

void team5() {
    int x = 5;
    for (int i = 0; i < x; i++) { // Logical Error: Should iterate 4 times
        cout << x + i << endl;
    }
    cout << "Sum: " << (x + i << endl; // Syntax Error: Missing closing parenthesis
    cout << "You qualify for the next round!" << endl;
}

int main() {
    team5();
    return 0;
}

File: python codes/code_5.py
def team5():
    x = 5
    for i in range(x):  # Logical Error: Should iterate only 4 times
        print(x + i)
    print(f"Sum of i and x: {sum(x + i for i in range(x)})  # Syntax Error: Missing closing parenthesis
    print("You qualify for the next round!")

team5()
